### PR TITLE
cppcheck: new package

### PR DESCRIPTION
**Pull request by ys-nuem**

### CODE REVIEW

File: mingw-w64-cppcheck/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+_realname=cppcheck
+pkgbase=mingw-w64-${_realname}
+pkgname=("${MINGW_PACKAGE_PREFIX}-${_realname}")
+pkgver=1.72
+pkgrel=1
+pkgdesc="static analysis of C/C++ code (mingw-w64)"
+arch=('any')
+url="http://cppcheck.sourceforge.net/"
+license=('GPL3')
+depends=("${MINGW_PACKAGE_PREFIX}-qt5"
```
Comment (Alexpux) [inline mingw-w64-cppcheck/PKGBUILD]: It really need Qt5?

Comment (ys-nuem) [inline mingw-w64-cppcheck/PKGBUILD]: It might not need...
I'm trying to build under Qt4.

Comment (ys-nuem) [inline mingw-w64-cppcheck/PKGBUILD]: It was able to build with Qt4.
I'll fix the dependency.

**Comment by ys-nuem** [comment]
Hmm... I added my local log file by mistake. Really sorry.

**Comment by ys-nuem** [comment]
Done.

**Comment by ys-nuem** [comment]
Ok, squash and push after checking in my own environment.

**Comment by ys-nuem** [comment]
Rebased PR to follow the latest master
I am sorry to have caused trouble.

**Pull request merged**